ReactWindows/ReactNative.Net46/DevSupport/WebSocketJavaScriptExecutor.cs

```diff
@@ -0,0 +1,253 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using ReactNative.Bridge;
+using ReactNative.Collections;
+using SocketMessaging;
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Runtime.ExceptionServices;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace ReactNative.DevSupport
+{
+    class WebSocketJavaScriptExecutor : IJavaScriptExecutor
```

Comment:
Unfortunately, no. I don't have any manual test case other than the fact that the interface is public unchanged and it compiles. :(